database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
@tannerwelsh When we were creating this schema there was debate about whether to use a foreign key or an implicit association. Can you speak to why we do want a foreign key here and not an implicit association?